CODE_OF_CONDUCT.md

```diff
@@ -60,9 +60,8 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-[mail](mailto:).
+[kobedo11@gmail.com](mailto:kobedo11@gmail.com).
```

Comment:
Replace to `the discord server`

Reply:
I would accept it being the discord server if there were more management and also more maturity.

Reply:
Can you please check that now?